studio/src/reactivity/store-controller.js

```diff
@@ -0,0 +1,34 @@
+import { LitElement } from 'lit';
+import { ReactiveStore } from './reactive-store.js';
+
+export default class StoreController {
+    host;
+    store;
+
+    value;
+
+    /**
+     *
+     * @param {LitElement} host
+     * @param {ReactiveStore} store
+     */
+    constructor(host, store) {
+        this.store = store;
+        this.value = store.get();
```

Comment:
why the initial value is get from store here and later at the line 31, the value in the store is not updated?

Reply:
In the controller I simply initialize the local state (`this.value`). At line 31, the value in the story is already updated, `updateValue` reacts to that, updating the local state.